.github/labels.yml

```diff
@@ -0,0 +1,219 @@
+  color: "0078D4"
+  description: Issue or feature relates specifically to Edge browser
+
+- name: "browser: any"
+  color: "6F42C1"
+  description: Issue or feature relates to any browser
+
+- name: "platform: android"
+  color: "3DDC84"
+  description: Issue or feature relates specifically to Android platform
+
+- name: "platform: iOS"
+  color: "007AFF"
+  description: Issue or feature relates specifically to iOS platform
+
+- name: "platform: web"
+  color: "E34F26"
+  description: Issue or feature relates specifically to web platform
+
+- name: "platform: linux"
+  color: "FCC624"
+  description: Issue or feature relates specifically to Linux platform
+
+- name: "platform: windows"
+  color: "0078D4"
+  description: Issue or feature relates specifically to Windows platform
+
+- name: "platform: mac"
+  color: "000000"
+  description: Issue or feature relates specifically to macOS platform
+
+- name: "platform: server"
+  color: "8B5CF6"
+  description: Issue or feature relates specifically to server platform
+
+- name: "fyi: qa"
+  color: "F39C12"
+  description: For information - Quality Assurance team
+
+- name: "fyi: architecture"
+  color: "E74C3C"
+  description: For information - Architecture team
+
+- name: "fyi: design"
+  color: "9B59B6"
+  description: For information - Design team
+
+- name: "fyi: product"
+  color: "27AE60"
+  description: For information - Product team
+
+- name: "fyi: engineering"
+  color: "3498DB"
+  description: For information - Engineering team
+
+- name: "fyi: face"
+  color: "0553B1"
+  description: For information - Frontend Catalyst Experience Team
+
+- name: "fyi: gatekeepers"
+  color: "D590CC"
+  description: For information - Catalyst App Backend, System Development & Integration Team
+
+- name: "fyi: cot"
+  color: "944e7e"
+  description: For information - Catalyst Ops & Tools Team
+
+- name: "fyi: hermes"
+  color: "bfd4f2"
+  description: For information - Hermes Backend, System Development & Integration Team
+
+- name: "rr: qa"
+  color: "D68910"
+  description: Required review - Quality Assurance team must approve before merge
+
+- name: "rr: architecture"
+  color: "C0392B"
+  description: Required review - Architecture team must approve before merge
+
+- name: "rr: design"
+  color: "7D3C98"
+  description: Required review - Design team must approve before merge
+
+- name: "rr: product"
+  color: "1E8449"
+  description: Required review - Product team must approve before merge
+
+- name: "rr: engineering"
+  color: "2E86C1"
+  description: Required review - Engineering team must approve before merge
+
+- name: "rr: face"
+  color: "033A80"
+  description: Required review by face team
+
+- name: "rr: gatekeepers"
+  color: "A569BD"
+  description: Required review by gatekeepers team
+
+- name: "rr: cot"
+  color: "6C3483"
+  description: Required review by cot team
+
+- name: "rr: hermes"
+  color: "85C1E9"
+  description: Required review by hermes team
+
+- name: refactor
```

Comment:
Do we really need a label for this?

Reply:
Yes, we needed to discuss this last week and agreed to have it.